README.md

```diff
@@ -209,6 +209,29 @@ trie.put('key', 'value', () => {
 
 ```
 
+## Examples (webpack.config.js)
+
+To bundle with webpack instead of browserify, resolve the `fs` module with the `graceful-fs` module.
+
+```js
+const path = require('path')
+
+module.exports = {
+  entry: './src/dat.js',
```

Comment:
Is this file specific to the project you'd be running?

Reply:
Yes.

```suggestion
  entry: './src/index.js',
```

To match https://webpack.js.org/#bundle-it